src/components/budget/budget.service.ts

```diff
@@ -445,7 +445,15 @@ export class BudgetService {
       ])
       .with('{value: props.value, property: type(r)} as prop, permList, node')
       .with('collect(prop) as propList, permList, node')
-      .return('propList, permList, node')
+      .match([
+        node('node'),
+        relation('out', '', 'organization', { active: true }),
+        node('organization', 'Organization', { active: true }),
+      ])
+      .with('propList, permList, node, organization.id as organizationId')
+      .return(
+        'propList + [{value: organizationId, property: "organization"}] as propList, permList, node'
+      )
```

Comment:
The extra with clause isn't needed.
```suggestion
      .return([
        'propList + [{value: organization.id, property: "organization"}] as propList',
        'permList',
        'node',
      ])
```

Reply:
Good catch! Thanks.